README.md

```diff
@@ -45,9 +45,17 @@ are being created:
 # Cleanup
 
 In cases where the beaker process is killed before finishing, it may leave resources in Hetzner cloud. These will need to be manually deleted.
-
 Look for servers in your project named exactly as the ones in your beaker host configuration and SSH keys with names beginning with `Beaker-`.
 
+Every created cloud instance gets a label `delete_vm_after: 1698792887`. By
+default this is the unit timestamp during VM creation + an hour.
+
+You can modify the default of an hour by setting the `BEAKER_HCLOUD_DELETE_VM_AFTER`
+environment variable to any positive integer. It will be interpreted as hours.
```

Comment:
I'd consider making it seconds, so there is a bit more fine grained control. The `delete_vm_after` is also in seconds (since it's a UNIX timestamp), so it'd be consistent.

Reply:
I switched to seconds.